backend/development.ini.sample

```diff
@@ -187,10 +187,14 @@ preview_cache_dir = %(basic_setup.preview_cache_dir)s
 ### Session ###
 
 ### pyramid_beaker parameters ###
+
+# "File" storage config :
 session.type = file
-; session.data_dir = %(here)s/sessions_data
 session.data_dir = %(basic_setup.sessions_data_root_dir)s/sessions_data
-; session.lock_dir = %(here)s/sessions_data
+## "Redis" storage config sample:
+# session.type = ext:redis
+# note: redis db name are integer (by default 16 database are available from 0 to 15 db)
+# session.url = redis://localhost:6379/0
```

Comment:
We have make a big job on development.ini file Tracim 2.7, for standardize parameter. See top of development.ini file.

Reply:
i just readd deleted default value, i do think it's now ok (unless we decided to change the default session method).

Reply:
Ok, thanks.